examples/mutag_gin_equib.py

```diff
@@ -0,0 +1,100 @@
+import os.path as osp
+
+import torch
+import torch.nn.functional as F
+from torch.nn import BatchNorm1d, Linear, ReLU, Sequential
+
+from torch_geometric.datasets import TUDataset
+from torch_geometric.loader import DataLoader
+from torch_geometric.nn import EquilibriumAggregation, GINConv
+
+path = osp.join(osp.dirname(osp.realpath(__file__)), '..', 'data', 'TU')
+dataset = TUDataset(path, name='MUTAG')
+
+train_dataset = dataset[len(dataset) // 10:]
+test_dataset = dataset[:len(dataset) // 10]
+
+train_loader = DataLoader(train_dataset, batch_size=1)
+test_loader = DataLoader(test_dataset, batch_size=1)
+
+
+class Net(torch.nn.Module):
+    def __init__(self, in_channels, dim, out_channels):
+        super().__init__()
+
+        self.conv1 = GINConv(
```

Comment:
Replace with `torch.nn.models.GIN`.

Reply:
Yeah sorry this is just lifted from the previous example for now